scripts/build_example

```diff
@@ -81,6 +85,21 @@ case "${language}" in
     exit 1;;
 esac
 
+if [[ "${docker_image}" == true ]]; then
+  if [[ "${language}" != "rust" ]]; then
+    echo "Docker images are only supported for Rust modules"
+    exit 1
+  fi
+
+  readonly DOCKER_BAZEL_BUILD_FLAGS=(
+    '--symlink_prefix=bazel-docker-'
```

Comment:
Why do we need a different prefix? We should use the same that we normally use for `config=enc-sim`.